src/special/npm-scirpts.js

```diff
@@ -0,0 +1,12 @@
+import * as path from 'path';
```

Comment:
@rumpl You made a typo for the name of the file

Reply:
I'm tired... Thanks